src/SW_Domain.c

```diff
@@ -156,6 +156,57 @@ void SW_DOM_read(SW_DOMAIN* SW_Domain, LOG_INFO* LogInfo) {
                                 Time_get_lastdoy_y(SW_Domain->endyr) : 365;
                 fenddy = swTRUE;
                 break;
+            case 8: // Spinup Mode
+                y = atoi(value);
+                
+                if (y != 1 || y != 2) {
```

Comment:
The check against bad user input for spinup mode `y != 1 || y != 2` is always true -- see "warning: overlapping comparisons always evaluate to true [-Wtautological-overlap-compare]"
For example, if y = 1, then y != 2 is true; if y = 2, then y != 1 is true; if y = 3, then both y != 1 and y != 2 are true.

It appears that the desired logic should be `!(y == 1 || y == 2)` or (equivalently) `y != 1 && y != 2`